### PR TITLE
Extend tools to print information containers information

### PR DESCRIPTION
**Pull request by mauriciovasquezbernal**
This PR extends the tools supported by Inspektor Gadget to print more information about the containers where the events happen. 

This logic uses an eBPF map that is created by the gadget tracer manager with all the information about the different containers running in the node. 

To be used with https://github.com/kinvolk/inspektor-gadget/pull/207.

### CODE REVIEW

File: tools/tcpconnect.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 from __future__ import print_function
 from bcc import BPF
-from bcc.containers import filter_by_containers
+from bcc.containers import filter_by_containers, ContainersMap, print_container_info
```
Comment (alban) [inline tools/tcpconnect.py:26]: Commitmsg has a typo: `tools/tcpconncet`

**Review by alban — COMMENTED** [review]
Thanks!

File: src/python/bcc/containers.py [hunk 2]
```diff
@@ -81,3 +103,54 @@ def filter_by_containers(args):
...
+
+    # Get the details of the container from the containers map.
+    def get_container(self, mntnsid):
+        key = ct.c_ulonglong(mntnsid)
+        containerC = ContainerC()
+        container = Container()
+
+        ret = lib.bpf_lookup_elem(self.map_fd, ct.byref(key), ct.byref(containerC))
+        if int(ret) != 0:
+            # TODO: raise?
```
[the remark this answers is not in the document]
Comment (mauriciovasquezbernal) [reply src/python/bcc/containers.py:149]: Makes sense. I removed the TODO and just return a the object with the fields set to `<>`

File: src/python/bcc/containers.py [hunk 3]
```diff
@@ -81,3 +103,54 @@ def filter_by_containers(args):
...
+# keep synchronized with definition in gadget tracer manager
+BUFFER_SIZE = 256
+
+class ContainerC(ct.Structure):
+    _fields_ = [
+        ("ContainerID", ct.c_char*BUFFER_SIZE ),
+        ("KubernetesNamespace", ct.c_char*BUFFER_SIZE),
+        ("KubernetesPod", ct.c_char*BUFFER_SIZE),
+        ("KubernetesContainer", ct.c_char*BUFFER_SIZE),
+    ]
```
[the remark this answers is not in the document]
Comment (mauriciovasquezbernal) [reply src/python/bcc/containers.py:113]: I think we can keep them in this repository for the time being. Later one we'll find out what's the best approach. What do you think?

[the remark this answers is not in the document]
Comment (alban) [reply src/python/bcc/containers.py:113]: Fine for me

**Review by alban — APPROVED** [review]
LGTM

**Pull request merged**